apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -51,6 +51,19 @@ const Icon = () => {
   return <Ellipse {...headerIconsSettings} />;
 };
 
+/** Get the title to show based on the specified color mode preference **/
+const getThemeTitleByColorMode = (mode) => {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
const getThemeTitleByColorMode = (mode: string) => {
```